src/drive/web/modules/views/Sharings/index.jsx

```diff
@@ -32,7 +32,7 @@ import {
 import { useFilesQueryWithPath } from 'drive/web/modules/views/hooks'
 import { useExtraColumns } from 'drive/web/modules/certifications/useExtraColumns'
 import { makeExtraColumnsNamesFromMedia } from 'drive/web/modules/certifications'
-import FileListRowsPlaceholder from 'drive/web/modules/filelist/FileListRowsPlaceholder'
+import Spinner from 'cozy-ui/react/Spinner'
```

Comment:
On a l'habitude de regrouper les import par thème, dans cet ordre : 
- les import react et libs
- les import libs cozy
- les import interne à l'app
- les imports de styles

avec une séparation entre chaque :)

Reply:
Je suis revenu sur `FileListRowsPlaceholder`, mais merci de l'info, c'est noté :)